src/notifier/index.js

```diff
@@ -0,0 +1,107 @@
+import dotenv from 'dotenv';
+import config from 'config';
+dotenv.config();
+
+import sendInBlue from 'sib-api-v3-sdk';
+const defaultClient = sendInBlue.ApiClient.instance;
+
+const authentication = defaultClient.authentications['api-key'];
+authentication.apiKey = process.env.SENDINBLUE_API_KEY;
+
+const apiInstance = new sendInBlue.SMTPApi();
+const contactsInstance = new sendInBlue.ContactsApi();
+
+const ADMINISTRATORS_LIST_ID = config.get('notifier.sendInBlue.administratorsListId');
+const UPDATES_LIST_ID = config.get('notifier.sendInBlue.updatesListId');
+const UPDATE_TEMPLATE_ID = config.get('notifier.sendInBlue.updateTemplateId');
+const ERROR_TEMPLATE_ID = config.get('notifier.sendInBlue.errorTemplateId');
+const BASE_URL = config.get('notifier.sendInBlue.baseUrl');
+
+const serviceProvidersMailingLists = {};
+let serviceProviders;
+let documentTypes;
+
+export function init(passedServiceProviders, passedDocumentTypes) {
+  serviceProviders = passedServiceProviders;
+  documentTypes = passedDocumentTypes;
+}
```

Comment:
If we're introducing class usage in this app, I believe this clearly calls for using it here as well 🙂